### DISCUSSION TITLE
ReprapPro Ormerod 1 design files

### DISCUSSION THREAD

**Discussion by kenroma6946** [community/community#124991] [Repositories]
### Select Topic Area

Question

### Body

I have an old Reprappro Ormerod 1 printer which I would like to restore for the odd hobby project.  Where can I find the download link for all the design files?

**Comment by queenofcorgis** [comment]
Hi @kenroma6946 ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as the Reprappro Ormerod 1 printer's community. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
